geoapps/base/application.py

```diff
@@ -301,10 +302,15 @@ def get_output_workspace(self, workpath: str = "./", name: str = "Temp.geoh5"):
         """
         Create an active workspace with check for GA monitoring directory
         """
+        timestamp = f"_{time.time():.0f}"
         if not name.endswith(".geoh5"):
-            name += ".geoh5"
+            name += timestamp + ".geoh5"
+        else:
+            name.replace(".geoh5", timestamp + ".geoh5")
 
-        workspace = Workspace(path.join(workpath, name))
+        h5file = path.join(workpath, name)
+        h5file = existing_file_incrementer(h5file)
```

Comment:
I don't think we will ever need this since we are adding a timestamp...

Reply:
What issue will the timestamp come in with though... I tested 508 yesterday and was still getting double run errors